maven/bnd-testing-maven-plugin/src/main/java/aQute/bnd/maven/testing/plugin/TestingMojo.java

```diff
@@ -103,94 +93,11 @@ private void testing(File runFile) throws MojoExecutionException, Exception, IOE
 			}
 
 			if (resolve) {
-				resolve(run);
+				run.resolve(failOnChanges);
```

Comment:
The replaced `resolve` method did not modify the bndrun file. It just set the computed runbundles in the Run object: `run.setRunBundles(runBundles)`.  So we need to consider that here. Not sure what the right thing to do it, but I suspect that running test should not modify a source file. The export mojo modifies it as well. In the gradle plugin there is an explicit resolve task to modify the bndrun file. export only exports the -runbundles present.